.gitlab-ci.yml

```diff
@@ -144,18 +157,20 @@ before_script:
 'alpine 3.5 2.1':
   <<: *build_definition
   variables:
-    OS: 'alpine_3.5'
+    OS: 'alpine'
+    OS_VER: '3.9'
     TAG: '2.1'
     VER: '2.1'
-    DVER: '2.2'
+    DVER: '1.x'
```

Comment:
What is DVER? Why it is changed here?

Reply:
DVER means dockerfile version, it shows the dockerfile to be used for building.

Reply:
The commit is named as 'Bump alpine images version'. Why DVER is changed here from 2.2 to 1.x?

Reply:
Moved to standalone PR.

Reply:
But it still present here.